### ISSUE TITLE
[Component] Icon should execept valid Iconnames

### ISSUE THREAD

**Issue by siluri** [Semantic-Org/Semantic-UI-React#2694]
### Version
latest

### Testcase
```jsx
<Button inverted icon>
    <Icon name="lightbulb outline" />
</Button>
```
```shell
Warning: Failed prop type: Invalid prop `name` of value `lightbulb outline` supplied to `Icon`.

Instead of `lightbulb outline`, did you mean:
  - lightbulb
  - like outline
  - building outline
...
```

### Solution
i guess, u must add that icon in the [static list](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/lib/SUI.js):
```js
export const ICON_ALIASES = [  ... 'lightbulb', 'lightbulb outline', ... ]
```

**Comment by brianespinosa** [comment]
Duplicate of #2550. The new icon names are not supported yet with PropType validation. You can still use them and they will work if you want to use the latest CSS and you are not blocked by the other issues in #2550. Or you can also add `lightbulb outline` as a className prop on your `<Icon />` component if you want to avoid the validation errors in development.
